packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -5,7 +5,43 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { Event, EventEmitter, TreeDataProvider, TreeItem } from 'vscode';
+import {
+  Event,
+  EventEmitter,
+  TreeDataProvider,
+  TreeItem,
+  TreeItemCollapsibleState
+} from 'vscode';
+
+import {
+  APEX_EXECUTION_OVERLAY_ACTION_ACTION_SCRIPT,
+  APEX_EXECUTION_OVERLAY_ACTION_ACTION_SCRIPT_TYPE,
+  APEX_EXECUTION_OVERLAY_ACTION_EXECUTABLE_ENTITY_NAME,
+  APEX_EXECUTION_OVERLAY_ACTION_IS_DUMPING_HEAP,
+  APEX_EXECUTION_OVERLAY_ACTION_ITERATION,
+  APEX_EXECUTION_OVERLAY_ACTION_LINE,
+  SFDC_TRIGGER
+} from '../constants';
+
+import { CheckpointUtil } from './checkpointUtil';
+
+import { DebugProtocol } from 'vscode-debugprotocol/lib/debugProtocol';
```

Comment:
The ordering of the imports seems off. Tslint usually would've complained to move `from 'vscode-debugprotocol/lib/debugProtocol'` to be below `from 'vscode'`. Is it because you have a new line in between imports? Other files don't do that.

Reply:
Yeah, you're right, it was the newline.